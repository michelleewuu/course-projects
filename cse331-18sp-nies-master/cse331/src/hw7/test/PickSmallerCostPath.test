# Test find path on picking the minimum-cost path from two
# available paths

# Create a graph with two Nodes and two Edges with different cost between them
CreateGraph simpleGraph
AddNode simpleGraph node1
AddNode simpleGraph node2
AddEdge simpleGraph node1 node2 1
AddEdge simpleGraph node1 node2 1.001

# Find the minimum-cost path from node1 to node2
FindPath simpleGraph node1 node2