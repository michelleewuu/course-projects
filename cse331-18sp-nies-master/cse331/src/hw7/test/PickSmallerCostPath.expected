# Test find path on picking the minimum-cost path from two
# available paths

# Create a graph with two Nodes and two Edges with different cost between them
created graph simpleGraph
added node node1 to simpleGraph 
added node node2 to simpleGraph 
added edge 1.000 from node1 to node2 in simpleGraph
added edge 1.001 from node1 to node2 in simpleGraph

# Find the minimum-cost path from node1 to node2
path from node1 to node2:
node1 to node2 with weight 1.000
total cost: 1.000